tests/test_bin.py

```diff
@@ -223,18 +228,22 @@ def test_hy2py():
                 if f == "py35_only_tests.hy" and not PY35:
                     continue
                 i += 1
-                output, err = run_cmd("hy2py -s -a " +
-                                      os.path.join(dirpath, f))
+                output, err = run_cmd("hy2py -s -a", args=[os.path.join(dirpath, f)])
```

Comment:
Sorry for the delays, but...adding an extra argument `args=` that's really...odd (you can pass arguments through both `cmd` *and* `args`) for just *one* test case feels really wrong. A simpler change would be:

```python
output, err = run_cmd("hy2py -s -a " + shlex.quote(os.path.join(dirpath, f)))
```

(Of course, using `pipes.quote` instead on Python 2.) Then you can drop the whole `args` mess.

Reply:
I thought it might get used by future tests though.